### ISSUE TITLE
position probability plot: improve color map

### ISSUE THREAD

**Issue by magland** [magland/sortingview#186]
Waiting for a high entropy example.

**Comment by edeno** [comment]
How hard would it be for the user to be able to specify the colormap?

**Comment by jsoules** [comment]
Depends on what you want. We could allow the user to specify color map at figure create time, but adding a UI element to change it dynamically would probably be more trouble than it's worth. Additionally, choosing a good color map seems like a lot to ask for every user.

Another possibility is to use one of the perceptually uniform color maps (like we're doing for the track animation view). There are currently three or four of those supported, and while the choice is hard-coded right now, exposing it via code wouldn't be too hard.

**Comment by edeno** [comment]
Yeah, I didn't mean for it to be dynamically choosable. I was just thinking of the options matplotlib provides: https://matplotlib.org/stable/tutorials/colors/colormaps.html which indeed could just be the perceptually uniform ones.

**Comment by jsoules** [comment]
I think we currently have imported magma, plasma, inferno, and viridis from the R viridis color maps, if that works. Track animation is currently set to magma I believe (with some amplification--for that view I use (value in (0, 255))/125, so roughly doubled its actual value, since the values tend to be very low overall.)

**Comment by jsoules** [comment]
https://cran.r-project.org/web/packages/viridis/vignettes/intro-to-viridis.html

**Comment by edeno** [comment]
I think that should be sufficient.
